pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -96,6 +96,11 @@ pub mod pallet {
 		/// Maximum size of an incoming message.
 		#[pallet::constant]
 		type MaxIncomingMessageSize: Get<u32>;
+
+		/// The sender account that will be used in the OutboundQueue
+		/// implementation.
+		#[pallet::constant]
+		type Sender: Get<Self::AccountId>;
```

Comment:
I think the name and the intention are somewhat ambiguous. This is used to specify an account that will be paying for fees, a `FeePayer`. It's not specifying who is sending the message per se. This is a naming detail but getting these small details slightly tweaked and accumulated make a difference in the big picture mental model we make out of these dozens and complex pallets 🙏

Reply:
In the case of XCM - yes, it's a fee payer, in the EVM case - it's both the fee payer and the sender of the message. As far as routers are concerned, it's also sender. IMO calling it a `FeePayer` is even more confusing.